src/basic/conf-files.c

```diff
@@ -178,6 +178,41 @@ int conf_files_list_strv(
         return get_sorted_files_from_hashmap(fh, ret);
 }
 
+int conf_files_list_strv_at(
```

Comment:
`conf_files_list_strv()` can be modified to just open the root directory and then call into this function.